framework/db/src/main/java/com/cloud/utils/db/TransactionLegacy.java

```diff
@@ -95,6 +98,8 @@ public class TransactionLegacy implements Closeable {
         }
     }
 
+    private static final String CONNECTION_POOL_LIB_DBCP = "dbcp";
```

Comment:
@shwstppr minor - can we change the default to hikaricp, so when not defined hikaricp is used; otherwise we honour one of which is defined in db.properties ?

Reply:
@rohityadavcloud we are already doing that in `createDataSource` and `getDefaultDataSource`